packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js

```diff
@@ -149,18 +152,21 @@ export const US_GUIDES = [
   {
     title: 'dedicated_clouds_onboarding_us_guide1_title',
     description: 'dedicated_clouds_onboarding_us_guide1_description',
+    id: 'How-to-Access-the-NSX-Interface',
```

Comment:
if duable, better to have same naming convention regarding others id.
i guess using lowercase id is better.

Reply:
It's applied.